packages/build/src/core/constants.js

```diff
@@ -129,6 +129,10 @@ const normalizePath = function (path, buildDir, key) {
   return pathA
 }
 
+const isEmptyValue = function (path) {
+  return path === undefined || path === ''
```

Comment:
Some configuration properties can be set to an empty string by plugins to reset them.
For example, setting the build command or Functions directory to an empty string will skip them.